### DISCUSSION TITLE
SFTP provider throwing import error for nacl

### DISCUSSION THREAD

**Discussion by bab014** [apache/airflow#22288] [General] [answered]
### Apache Airflow Provider(s)

sftp

### Versions of Apache Airflow Providers

apache-airflow-providers-sftp==2.5.0

### Apache Airflow version

2.2.4 (latest released)

### Operating System

Amazon Linux AMI (rhel fedora)

### Deployment

Other

### Deployment details

Just a basic install onto an Ec2 instance

### What happened

Tried using the SFTPSensor. DAG was unable to import 
```python
Broken DAG: [/rhome/bab084/airflow/dags/grab_files.py] Traceback (most recent call last):
  File "/rhome/bab084/.local/lib/python3.7/site-packages/nacl/bindings/__init__.py", line 16, in <module>
    from nacl.bindings.crypto_aead import (
  File "/rhome/bab084/.local/lib/python3.7/site-packages/nacl/bindings/crypto_aead.py", line 17, in <module>
    from nacl._sodium import ffi, lib
ImportError: /rhome/bab084/.local/lib/python3.7/site-packages/nacl/_sodium.abi3.so: undefined symbol: crypto_core_ed25519_nonreducedscalarbytes
```

### What you think should happen instead

I should be able to import the DAG without the error occuring

### How to reproduce

simply
```python
from airflow import DAG
from airflow.providers.sftp.sensors.sftp import SFTPSensor

from datetime import datetime


default_args = {
    'depends_on_past': False,
    'start_date': datetime(2022,3,8)
}

with DAG(
    'quad_graphics_file_transfer',
    default_args=default_args,
    catchup=False,
    schedule_interval='@weekly',
) as dag:

    # checking if files exist
    connect_to_server = SFTPSensor(
        task_id='check-server-conn',
        sftp_conn_id='connection',
        path='/mft_prd/QuadGraphics/Atlas/incoming'
    )
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Reply by potiuk** [reply]
This is clearly a problem with your environment not SFTP provider. You have badly installed libraries which have nothing do with Airflow.. I recommend you to start from scratch.

Plese don't report such things as "Airfllow" bugs.

**Reply by bab014** [reply]
I'm not sure that's the issue. I followed the documentation. 

Aside from a total teardown is there anything you would recommend?

**Reply by potiuk** [reply]
Try google for similar errors ?

**Comment by bab014** [comment, marked as answer]
I tried that first, would not have opened an issue without trying Google first. 

I'm going to close this and just rebuild.
